tools/scripts/release/crate-bump.sh

```diff
@@ -88,11 +88,15 @@ while [[ "$updated_crates" != "$recently_updated_crates" ]]; do
     echo y | cargo release "$version" --config tools/scripts/release/release.toml --no-push --no-publish --no-tag --no-dev-version --package "$name" --execute
   done
 
-  recently_updated_crates=$updated_crates
+  unset recently_updated_crates
```

Comment:
feels strange to unset then set it again immediately after?

Reply:
We are trying to handle scenarios like below
- `ockam_core` code was updated by developer, so it version gets bumped by the script
- `ockam_core` is a transitive dependency of `ockam` crate so we have to bump the `version` of `ockam_core` in `ockam` Cargo.toml file and also `ockam` version itself (since there was an update to the crate which is publishable).

To ensure we bump all updated crates, we need to track crates that were bumped due to code changes and their transitive deps

Reply:
@polvorin can you please re-review? Apologies.

Reply:
I mean,   what I don't get is the difference between
```
    unset recently_updated_crates
    recently_updated_crates="${updated_crates[@]}"
```
that the script does,  vs just

```
    recently_updated_crates="${updated_crates[@]}"
```

unless there is something I'm not seeing :man_shrugging:   .  If it is needed for something non-obvious lets put a comment on it noting why we need to unset before immediately setting it again.

Reply:
Works better. Thanks.

Reply:
Please re-review